src/Kafka/ManageKafkaPermissions/components/ResourceTypeOperationFilter/ResourceTypeOperationFilter.tsx

```diff
@@ -350,7 +358,7 @@ export const ResourceTypeOperationFilter: React.VFC<
   const transactionalIdMapped = transactionalIdOptions.map(mapTree);
 
   const menu = (
-    <Menu ref={menuRef}>
+    <Menu ref={menuRef as React.RefObject<HTMLDivElement>}>
```

Comment:
Try not to use `as` if you can. The proper way to specify the ref with Typescript would be:
`const toggleRef = useRef<HTMLButtonElement>(null);`

Note the `null`, it should fix the error.